47.md

```diff
@@ -373,6 +373,34 @@ Response:
 }
 ```
 
+### `get_budget`
+
+Budgets can be used to limit the amount of funds that can be spent by a connection in a given time period.
+Budgets are optional and can be set by the user if their wallet supports it. Support for this command indicates
+that the wallet supports user-configured budgets. The budget will be reset at the `renews_at` timestamp.
+
+Request:
+```jsonc
+{
+    "method": "get_budget",
+    "params": {
+    }
+}
+```
+
+Response:
+```jsonc
+{
+    "result_type": "get_budget",
+    "result": {
+        "used_budget": 10000,
+        "total_budget": 100000,
```

Comment:
Are the units still msats? I'm okay with not having the unit in the name, but it should be stated somewhere in the documentation.

Reply:
ah, thanks for the catch - I'd removed that comment when I switched to `_msats` for the field name, but forgot to add it back. Done